packages/react-components/src/components/timegraph-output-component.tsx

```diff
@@ -717,14 +733,51 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
     }
 
     private getRowStyle(row?: TimelineChart.TimeGraphRowModel) {
+
+        let backgroundColor = 0x979797;
+
+        if (row?.selected) {
+            const colorString = getComputedStyle(document.body).getPropertyValue('--theia-selection-background');
+            const colorNumber = this.convertColorStringToHexNumber(colorString);
+            backgroundColor = colorNumber > 0 ? colorNumber : backgroundColor;
+        }
+
         return {
-            backgroundColor: 0x979797,
-            backgroundOpacity: row?.selected ? 0.1 : 0,
+            backgroundColor,
+            backgroundOpacity: row?.selected ? 1 : 0,
             lineColor: this.props.backgroundTheme === 'light' ? 0xD3D3D3 : 0x3F4146,
             lineThickness: 1
         };
     }
 
+    private convertColorStringToHexNumber(rgb: string): number {
```

Comment:
shouldn't this kind of method part of a utility class? Maybe have the util in directory `packages/base/src/utils`.

Reply:
Moved this to an exported function in that directory.